merlin/models/tf/inputs/embedding.py

```diff
@@ -414,7 +426,14 @@ def Embeddings(
                 combiner = sequence_combiner
 
         embedding_size = embedding_dims.get(col.name, embedding_dim_default)
-        tables[col.name] = EmbeddingTable(embedding_size, col, combiner=combiner, **kwargs)
+        tables[col.name] = EmbeddingTable(
+            embedding_size,
+            col,
+            combiner=combiner,
+            trainable=col not in non_trainable,
+            embeddings_initializer=embeddings_initializers.get(col.name, "uniform"),
```

Comment:
Need to handle the case where embeddings_initializers is None or not a dict. Similar to this part in the EmbeddingFeatures

https://github.com/NVIDIA-Merlin/models/blob/c63daf65efc653cd84cc32ba64c8ce8f110af220/merlin/models/tf/inputs/embedding.py#L629-L632

Reply:
makes sense.